lib/qa/authorities/linked_data/rdf_helper.rb

```diff
@@ -0,0 +1,82 @@
+require 'rdf'
+
+# Encapsulates processing of RDF results returned by the linked data authority.  This is used exclussively by Qa::Authorities::LinkedData::GenericAuthority
+# @see Qa::Authorities::LinkedData::GenericAuthority
+module Qa::Authorities
+  module LinkedData
+    module RdfHelper
```

Comment:
@hackmastera that was an old comment and is no longer relevant after we worked on it at the dev congress.